examples/showcase/worksheets.ptx

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<!-- This file is part of the PreTeXt Showcase article. -->
+<!--                                                    -->
+<!-- Copyright (©) 2021 The PreTeXt Organization.       -->
+<!-- See the README file for contributing guidelines.   -->
+
+<section xml:id="worksheets">
+  <title>Worksheets</title>
+  
+  <worksheet>
+    <title>A Geometric Prelude</title>
+    <!-- <author>Dave Rosoff</author> -->
+    <objectives xml:id="objectives">
+      <ul>
+        <li>Practice visualizing vector addition</li>
+        <li>Use vectors without explicit coordinates</li>
+      </ul>
+    </objectives>
+    <introduction>
+      <p>It was known to Euclid, and probably earlier, that the midpoints of the sides of any quadrilateral all lie in the same plane (even if the vertices of the quadrilateral do not). In fact, these midpoints are the vertices of a parallelogram, as pictured in <xref ref="figure-midpoints-of-quadrilateral" text="type-global"/>.</p>
+      <sidebyside width="30%">
+        <figure xml:id="figure-midpoints-of-quadrilateral">
+          <caption>The midpoints of the sides of a quadrilateral are the vertices of a parallelogram.</caption>
+          <image xml:id="worksheet-midpoints-of-quadrilateral">
+            <latex-image>
+              \begin{tikzpicture}[xscale=0.8, yscale=0.8]
+              \draw[style={black, ultra thick}] (0,0) -- (5,0) -- (4,4) -- (2,5) -- (0,0);
+              \draw[style={black, dashed, very thick}] (2.5, 0) -- (4.5, 2) -- (3, 4.5) -- (1, 2.5) -- (2.5, 0);
+              \end{tikzpicture}
+            </latex-image>
+          </image>
+        </figure>
+        <figure xml:id="figure-triangle-cyclic-vectors">
+          <caption>The sides of a triangle presented as vectors.</caption>
+          <image xml:id="worksheet-triangle-cyclic-vectors">
+            <latex-image>
+              \begin{tikzpicture}[xscale=1.5, yscale=1.5]
+              \draw[->,>=latex, style={black, thick}] (0,0) -- (2.30 , 0) node [below] {$\vec{A}$} --  (3,0);
+              \draw[->,>=latex, style={black, thick}] (3,0) -- (1.47, 1.53) node [above right =1mm] {$\vec{B}$} -- (1,2);
+              \draw[->,>=latex, style={black, thick}] (1,2) -- (0.23 , 0.46) node [above left=1mm] {$\vec{C}$} -- (0,0);
+              % \draw[->,>=latex, style={black,semithick,dashed}] (1,2) -- (7/6, 4/3) node
```

Comment:
Do we need all of the commented out lines here? The cleaner the Showcase Article is, the better.

Reply:
I prefer not to have "commented out" stuff in source.  It confuses searches for problems, etc.  And...it can be recalled from the git history.  Will look locally in a minute.